src/content/1.7/modules/testing/basic-checks.md

```diff
@@ -86,3 +86,34 @@ Using cache file ".php_cs.cache".
 
 Checked all files in 0.085 seconds, 12.000 MB memory used
 ```
+
+## License headers
+
+When the extension is open-source, sharing the license is one important task.
+
+[Header-Stamp](https://github.com/PrestaShopCorp/header-stamp) is a tool that applies a header on all compatible files (PHP, JS...).
```

Comment:
It should be in the PrestaShop org instead :thinking:

Reply:
This is not something @Quetzacoalt91 can control 😛 .

Reply:
:grimacing:

Reply:
@PierreRambaud I think we can merge the PR. When the project https://github.com/PrestaShopCorp/header-stamp is moved from PrestaShopCorp to PrestaShop, GitHub will take care of redirecting links?

Reply:
It usually does, yes. The opposite happened with https://github.com/PrestaShop/ps_checkout, you'll see the redirection is still valid.